### PR TITLE
Loading Raft on Catapult shows the correct icon, Raft blocks players from using actions

### PR DESCRIPTION
**Pull request by mugg91**
## Status

- **READY**: this PR is (to the best of your knowledge) ready to be incorporated into the game.

## Description

This is an addition/enhancement to [Pull Request #1215](https://github.com/transhumandesign/kag-base/pull/1215). You need to apply PR #1215 and then this to see the changes.

There are two changes. Please let me know what you think.

1)
Loading Raft on Catapult now shows the correct icon.
To do this, I edited VehicleIcons.png to add a small Raft sprite, replacing frame 7. (I did so because I think frame 7 was unused or was a placeholder anyway?)

2)
Players carrying a Raft will now be unable to use actions like slash, shield, bow, grapple(...)
Dinghy already has this implemented and I felt that Raft (which is supposed to be heavier weight) should implement this logic as well.
To do this, I added "BoatCommon.as" to "Raft.cfg".

## Steps to Test or Reproduce

- Try to load a Raft on a Catapult, notice it will use the correct sprite icon.
- Try to use actions while carrying a Raft, notice it will not work.

### CODE REVIEW

**Comment by eps0003** [comment]
It appears that you are manually uploading your changes to GitHub. This is not ideal since it could lead to human error when copying over your changes. Please look into how to set up a local dev environment so you can test and commit without needing to copy and paste. You can ask for help in #development channel on Discord if you are unsure how to do this.

**Comment by mugg91** [comment]
I'm not doing this on a highly professional level by any means. I'm using notepad++ and I'm having a separate dev install where I edit the files. If I come up with a solution I use the github file finder and copy paste the entire content of a file there. Before I PR, I copy paste the content of files back into my install and test if things are still going as expected. So there should not be a problem. I triple check everything. But I will take what you said into consideration for the future.

File: Entities/Vehicles/Boats/Raft.as [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Boat logic
+#include "GenericButtonCommon.as"
```
Comment (AsuMagic) [inline Entities/Vehicles/Boats/Raft.as:1]: This does not appear to be used?

**Review by AsuMagic — CHANGES_REQUESTED** [review]
`VehicleIcons.png` is now conflicting because of the outpost addition

File: Entities/Vehicles/Boats/Raft.as (hunk 1 above)
Comment (Gingerbeard5773) [reply Entities/Vehicles/Boats/Raft.as:1]: It is not used. This line can be removed safely.

**Review by Gingerbeard5773 — CHANGES_REQUESTED** [review]
VehicleIcons

The PR has an outdated image. Here's an updated image ^ that will not conflict.
I tested and found no other problems.

**Review by AsuMagic — APPROVED** [review]

**Pull request merged**